scheduler-jobs/deploy_data_pavics_jupyter.env

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+# To use in birdhouse-deploy, source this file in env.local.
+# This will generates jobs for the "deploy_data_specific_image" script found on the birdhouse/pavics-jupyter-base image.
+# A job will be created for each image found in the DOCKER_NOTEBOOK_IMAGES variable,
+# defined in the env.local from birdhouse-deploy.
+#
+# These jobs assume the "scheduler" component is enabled in birdhouse-deploy.
+#
+# Also, the image started by the job must contain the script "deploy_data_specific_image"
+# and a yaml config file "notebook_config.yml", and they must be placed on the root directory.
+# The script should be included in the pavics-jupyter-base image built from this repo.
+# Each specific images deriving from this base image should include its own "notebook_config.yml" file.
+##############################################################################
+
+DEPLOY_DATA_JOB_SCHEDULE="${AUTODEPLOY_NOTEBOOK_FREQUENCY}"
+DEPLOY_DATA_JOB_CONFIG="/notebook_config.yml"
+
+# Default path where notebooks are deployed on the image running the script
+NOTEBOOK_DEST_DIR=/tmp/notebooks
```

Comment:
Can you rename this to `DEPLOY_DATA_JOB_NOTEBOOK_DEST_DIR` to avoid potential future name clash.

Reply:
Oh wait a sec, this is the default value but where is it setting the real value?

Reply:
I guess I could remove the "default" in the comment here, as the variable is only set here, and is not really made to be overwritten anywhere else.

Reply:
So that value is by default `NOTEBOOK_DEST_DIR=/tmp/notebooks` but in reality we want it in `/data/jupyterhub_user_data/tutorial-notebooks/<image_name>`, so where is this real path being set?

Reply:
In this [code sample](https://github.com/bird-house/pavics-jupyter-base/blob/ebc86d5af1c919d78bc245effc269f5cea9686f8/scheduler-jobs/deploy_data_pavics_jupyter.env#L68), we mount the path such as `/data/jupyterhub_user_data/tutorial-notebooks/<image_name>` to the variable `DEPLOY_DATA_JOB_NOTEBOOK_DEST_DIR`

Reply:
> In this [code sample](https://github.com/bird-house/pavics-jupyter-base/blob/ebc86d5af1c919d78bc245effc269f5cea9686f8/scheduler-jobs/deploy_data_pavics_jupyter.env#L68), we mount the path such as `/data/jupyterhub_user_data/tutorial-notebooks/<image_name>` to the variable `DEPLOY_DATA_JOB_NOTEBOOK_DEST_DIR`

OHH, yes you're right.

But just to make understanding the code  easier, I'd suggest setting that var to the proper value then use `--volume ${DEPLOY_DATA_JOB_NOTEBOOK_DEST_DIR}:${DEPLOY_DATA_JOB_NOTEBOOK_DEST_DIR}:rw`

Reply:
Code updated with your suggestion :)